### PR TITLE
feat: Made responsive + added related articles

### PR DESCRIPTION
**Pull request by boudewijnbout**

### CODE REVIEW

**Comment by Finnvb** [comment]
lgtm m8

File: src/styles/pages/nieuwDetail.module.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .nieuwDetail {
     justify-content: center;
     max-width: 65rem;
-    margin: 10rem auto;
+    margin: 10rem auto 0 auto;
```
Comment (SCNMC) [inline src/styles/pages/nieuwDetail.module.css:7]: vgm is 10 rem auto voldoende

File: src/styles/pages/nieuwDetail.module.css [hunk 2]
```diff
@@ -62,7 +66,46 @@
...
-    margin: 1rem 0;
+    margin: 1.5rem 0;
+}
+
+/*--------------------------------------------------------------
+# Related articles
+--------------------------------------------------------------*/
+.relatedArticles h4 {
+    font-size: 1.1rem;
+    font-family: "Poppins", sans-serif;
```
Comment (SCNMC) [inline src/styles/pages/nieuwDetail.module.css:77]: change to css variable

File: src/styles/pages/nieuwDetail.module.css [hunk 3]
```diff
@@ -62,7 +66,46 @@
...
+    transition: transform 200ms;
+}
+
+.relatedArticles h5 {
+    margin-bottom: 0.75rem;
+}
+
+.relatedArticles p {
+    font-size: 0.95rem;
+    font-family: "Cormorant", sans-serif;
```
Comment (SCNMC) [inline src/styles/pages/nieuwDetail.module.css:108]: change to css variable

File: src/styles/theme.css [hunk 4]
```diff
@@ -5,12 +5,17 @@
...
     --white: #FFFFFF;
+
+    /* Borders */
     --border-color: #E5E5E5;
     --border-round: 10px;
+
+    /* Paddings */
     --padding-section: 1rem;
     --padding-section-desktop: 1.6rem 6.5rem;
     --font-cormorant: "Cormorant", sans-serif ;
```
Comment (SCNMC) [inline src/styles/theme.css:16]: dit hoort onder fonts

File: src/styles/pages/nieuwDetail.module.css [hunk 5]
```diff
@@ -85,14 +128,24 @@
...
+
+    .relatedArticles picture {
+        height: 12rem;
+    }
 }
 
 @media (min-width: 62rem) {
     .nieuwDetail {
-        margin: 11.5rem auto;
+        margin: 11.5rem auto 0 auto;
```
Comment (SCNMC) [inline src/styles/pages/nieuwDetail.module.css:148]: zelfde vgm werkt 11.5 rem auto prima

**Review by SCNMC — CHANGES_REQUESTED** [review]

File: src/styles/pages/nieuwDetail.module.css (hunk 1 above)
Comment (boudewijnbout) [reply src/styles/pages/nieuwDetail.module.css:7]: Besproken

File: src/styles/theme.css (hunk 4 above)
Comment (boudewijnbout) [reply src/styles/theme.css]: Changed

File: src/styles/pages/nieuwDetail.module.css (hunk 5 above)
Comment (boudewijnbout) [reply src/styles/pages/nieuwDetail.module.css:148]: Besproken

File: src/styles/pages/nieuwDetail.module.css (hunk 2 above)
Comment (boudewijnbout) [reply src/styles/pages/nieuwDetail.module.css]: Changed

**Review by SCNMC — APPROVED** [review]
lgtm

**Pull request merged**